gabrielaEnache/src/main/java/collections/Main.java

```diff
@@ -0,0 +1,64 @@
+package collections;
+
+import collections.store.*;
+import collections.store.Item;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static collections.store.Type.CLOTHES;
+import static collections.store.Type.TOY;
+
+public class Main {
+    public static void main(String[] args) {
+        Item toy = new Item("Bear", TOY);
+        Item cloth = new Item("T-shirt", CLOTHES);
+        Item toy1 = new Item("Car", TOY);
+        Item toy2 = new Item("Doll", TOY);
+
+        List<Item> shoppingList = new ArrayList<>();
+        shoppingList.add(toy);
+        shoppingList.add(cloth);
+
+        List<Item> shoppingList1 = new ArrayList<>();
+        shoppingList1.add(toy1);
+        shoppingList1.add(toy2);
+
+        Address ad1 = new Address("Moldova", "Chisinau", "Bulevardul Dacia");
+        Address ad2 = new Address("Moldova", "Balti", "Decebal 10");
+
+        User mihai = new User("Mihai");
+        mihai.addAddress("Home", ad1);
+
+        User ion = new User("Ion");
+        ion.addAddress("Work", ad2);
+        List<User> userList = new ArrayList<>(Arrays.asList(mihai, ion));
+
+        Order order = new Order(mihai, shoppingList);
+        Order order1 = new Order(ion, shoppingList1);
+
+        List<Order> orderList = new ArrayList<>(Arrays.asList(order, order1));
+
+        System.out.println("Filer orders by Item: " + filterOrdersByItemType(orderList, CLOTHES));
+        System.out.println("Filter users by City: " + filerUsersByCity(userList, "Chisinau"));
+    }
+
+    private static List<Order> filterOrdersByItemType(List<Order> listToSort, Type type) {
```

Comment:
You could define this method within Order class, also as static. Then it can be called within main method as following : Order.filterOrdersByItemType( listToSort, Type.clothes)

Reply:
also with static import

Reply:
Fixed